helm-charts/helm2/Makefile

```diff
@@ -0,0 +1,38 @@
+PROJECT_NAME=helm-charts
+
+include ../../Makefile.os
+
+RELEASE_VERSION ?= latest
+CHART_SEMANTIC_RELEASE_VERSION ?= $(shell cat ../../release.version | tr A-Z a-z)
+CHART_NAME=strimzi-kafka-operator
+CHART_PATH ?= ../helm2/strimzi-kafka-operator/
+CHART_RENDERED_TEMPLATES_TMP=../../target/charts
+CHART_RENDERED_TEMPLATES_CLUSTERROLES=../../cluster-operator/src/main/resources/cluster-roles/
+CHART_RENDERED_TEMPLATES_INSTALL=../../install/cluster-operator/
+HELM_CLI=helm2
+
+helm_clean:
+	rm -rf $(CHART_RENDERED_TEMPLATES_TMP)
+	rm -rf strimzi-$(RELEASE_VERSION)/charts/
+	rm -f $(CHART_NAME)-*.tgz
+
+helm_lint:
+	# Linting Helm Chart
+	$(HELM_CLI) lint --debug $(CHART_PATH)
+
+helm_pkg: helm_lint helm_install
```

Comment:
Should the `helm_install` be removed here? It looks like that target does not exist anymore. Or is it taken from somewhere else?

Reply:
It should. I kept `helm3` as default, so you are right.